Translation corrections (#483)
07th-mod/higurashi-console-arcs: Update/omot_025.txt

//rもちろん、それで銃弾が避けられるはずがない。俺に残されているのは、銃弾を身体のどこかに受けて…それから訪れる死までの時間だけだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "もちろん、それで銃弾が避けられるはずがない。俺に残されているのは、銃弾を身体のどこかに受けて…それから訪れる死までの時間だけだ。",

> 			NULL, "Of course, I can't dodge bullets. All it'll take is for the bullet to strike somewhere on my body... and then it's only a matter of time until death arrives.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Of course, I can't dodge bullets. All it takes is for the bullet to strike somewhere on my body... and then it's only a matter of time until death arrives.", Line_Normal);
```

//r死んだことのない俺には、…わかるはずもない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "死んだことのない俺には、…わかるはずもない。",

> 			NULL, "I've never died before... so there was no way I should have known.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I've never died before... so how could I know?", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/220100323", 256, TRUE);
	OutputLine(NULL, "「そ、そんなこと言ったって…！」",
			NULL, "\"S-saying that won't change anything...!\"", Line_Normal);
	ClearMessage();

//r思わず、弱気じみた悪態をついてしまう。どんなにあがいてみても、俺の全身はぴくりとも動いてくれないのだ。そんな状態で、一体何を期待しろというんだ？！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "思わず、弱気じみた悪態をついてしまう。どんなにあがいてみても、俺の全身はぴくりとも動いてくれないのだ。そんな状態で、一体何を期待しろというんだ？！",

> 			NULL, "I suddenly cursed faintheartedly. No matter how hard I tried, my entire body was unable to move. So what did she expect me to do in a situation like this?!", Line_Normal);

Reviewer: ```suggestion
			NULL, "I suddenly cursed faintheartedly. No matter how hard I tried, my entire body was unable to move. So, what did she expect me to do in a situation like this?!", Line_Normal);
```

	ClearMessage();

//圭一rvS14/01/220100324.「俺だって、こんなところで諦めたくない！kvS14/01/220100325.　みんなのことを守れずに死ぬなんて、真っ平ごめんだ！kvS14/01/220100326.　けどっ…身体が、動かないッ…！！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/220100324", 256, TRUE);
	OutputLine(NULL, "「俺だって、こんなところで諦めたくない！",

> 			NULL, "\"I don't want to give up at a time like this either!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I don't want to give up at a time like this, either!", Line_WaitForInput);
```

//rうつむき加減で踏む場所を誤ったのか、梨花ちゃんは浮石に足をとられて躓いて倒れる。そして膝をぶつけたのか、その場に顔をしかめてうずくまった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "うつむき加減で踏む場所を誤ったのか、梨花ちゃんは浮石に足をとられて躓いて倒れる。そして膝をぶつけたのか、その場に顔をしかめてうずくまった。",

> 			NULL, "She must've misstepped while looking down, because Rika-chan stumbled on a rock and fell to the ground. She must have hit her knee on something, because her face twisted with pain as she remained on the ground.", Line_Normal);

Reviewer: ```suggestion
			NULL, "She must've misstepped while looking down, because Rika-chan stumbled on a rock and fell to the ground. And her face twisted in pain as she remained on the ground, like she'd hit her knee on something.", Line_Normal);
```

//r小此木はレシーバーの向こうでヒステリックにがなりたてる上官に半ば辟易しながら、これも宮仕えとばかりに冷静に応対する。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "小此木はレシーバーの向こうでヒステリックにがなりたてる上官に半ば辟易しながら、これも宮仕えとばかりに冷静に応対する。",
			NULL, "Okonogi was half-sick of his commanding officer's hysterics on the other end of the receiver, but he remained calm and responded like a proper court servant.", Line_Normal);
	ClearMessage();

//r思った以上に抵抗勢力があっさり引き上げたことに少しだけ違和感があったが、おそらく兵装を整えるためだろうと思い直し、予定通りに神社へと集合する道を急いでいた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "思った以上に抵抗勢力があっさり引き上げたことに少しだけ違和感があったが、おそらく兵装を整えるためだろうと思い直し、予定通りに神社へと集合する道を急いでいた。",

> 			NULL, "He felt a bit uncomfortable that this greater resistance than originally expected had withdrawn so easily, but he figured they were getting their equipment in order, and so hurried down the road to the shrine as ordered.", Line_Normal);

Reviewer: ```suggestion
			NULL, "He felt a bit uncomfortable that this unexpectedly great resistance had withdrawn so easily, but he figured they were getting their equipment in order, and so hurried down the road to the shrine as ordered.", Line_Normal);
```
